vmslave-config/pipeline/vmslave_config.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+git diff --dirstat=files ${GIT_PREVIOUS_COMMIT} | grep "vmslave-config/"
```

Comment:
it's a weird behavior.
why use "GIT_PREVIOUS_COMMIT" as a judge condition to check if all other ENV are existing ?